### PR TITLE
Update HW requirement implementation links to match reality

### PR DESCRIPTION
**Pull request by skycastlelily**
Pull Request Checklist

* [x] update the specification

### CODE REVIEW

**Comment by happz** [comment]
@skycastlelily please, consider extending the subject. "Update spec files" can mean anything, and provides no pointer to what changed and why. Something like "Update HW requirement implementation links to match reality" would be way more helpful.

**Comment by skycastlelily** [comment]
Updated^^

File: spec/hardware/disk.fmf [hunk 1]
```diff
@@ -68,7 +68,8 @@ example:
 
 link:
   - implemented-by: /tmt/steps/provision/artemis.py
-    note: "``size`` only"
+    note: "``size`` ``model-name`` only"
```
Comment (therazix) [inline spec/hardware/disk.fmf:71]:
```suggestion
    note: "``size`` and ``model-name`` only"
```

**Review by therazix — CHANGES_REQUESTED** [review]

**Comment by thrix** [comment]
> Updated^^

@skycastlelily thx, I updated also MR description :) unfortunately it is not done automatically :(

File: spec/hardware/disk.fmf (hunk 1 above)
Comment (skycastlelily) [reply spec/hardware/disk.fmf]: Updated:)

**Comment by skycastlelily** [comment]
> > Updated^^
> 
> @skycastlelily thx, I updated also MR description :) unfortunately it is not done automatically :(

Thanks:)

**Review by thrix — APPROVED** [review]
LGTM

**Review by therazix — APPROVED** [review]

**Comment by psss** [comment]
Specification update --> full test not needed --> merging.

**Pull request merged**